### PR TITLE
[4.0] Add missing eye icons to installer css

### LINKED ISSUE #22963: [4.0] hide/show password icon missing and hover cursor wrong
### Steps to reproduce the issue

Attempt to install Joomla 4 from 

> commit 2a76a1d930d617773c80b2c9eee5d5605918e8e2 (HEAD -> 4.0-dev, origin/4.0-dev)

### Expected result

the ability to toggle the hide/show of passwords in the installer

### Actual result

a blank blue block with no icon in it, on hovering the cursor stays as a pointer instead of changing to a hand

screenshot 2018-11-08 at 05 25 06

### PR DESCRIPTION
**Pull request by PhilETaylor**
Closes #22963

### Summary of Changes

Added missing CSS for the eye and eye close icons (hide/show password icon) and set correct cursor

### Testing Instructions

checkout 4.0-dev
apply pr
npm ci
npm run build:css  (just in case) 
clear browser cache
load installation page, enter site name, click Setup Login data
On this next page note the new eye icon - click it

### Expected result

screenshot 2018-11-16 at 12 07 29

### Actual result

screenshot 2018-11-16 at 12 07 29

### Documentation Changes Required

None

### CODE REVIEW

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 85235d82ef92c1b9c84250c002ed2b6c35238f76<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23097">issues.joomla.org/tracker/joomla-cms/23097</a>.</sub>

File: installation/template/scss/template.scss [hunk 1]
```diff
@@ -281,3 +281,15 @@ caption {
 .badge-warning {
   color: #292b2c;
 }
+
+.fa-eye:before, .icon-eye-open:before, .icon-eye:before {
```
Comment (C-Lodder) [inline installation/template/scss/template.scss]: Shouldn't need to define this as it's already defined here: https://github.com/joomla/joomla-cms/blob/4.0-dev/build/media/system/scss/_icomoon.scss#L11

**Review by wilsonge — APPROVED** [review]

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 29aa3e20f2d116c7c761fe59122aba98da55510c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23097">issues.joomla.org/tracker/joomla-cms/23097</a>.</sub>

File: installation/template/scss/template.scss (hunk 1 above)
Comment (Quy) [inline installation/template/scss/template.scss:285]: `.fa-eye:before` and `.fa-eye-slash:before` are already declared elsewhere.

Comment (PhilETaylor) [reply installation/template/scss/template.scss:285]: Yes, but those instances are NOT loaded in the installation process!

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 29aa3e20f2d116c7c761fe59122aba98da55510c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23097">issues.joomla.org/tracker/joomla-cms/23097</a>.</sub>

**Comment by infograf768** [comment]
RTC
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23097">issues.joomla.org/tracker/joomla-cms/23097</a>.</sub>

File: installation/template/scss/template.scss (hunk 1 above)
Comment (Quy) [reply installation/template/scss/template.scss:285]: Fontawesome is imported on line 9.

Comment (infograf768) [reply installation/template/scss/template.scss:285]: It is indeed but the needed class is not. Weird.

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**